app/src/main/java/org/exoplatform/fragment/PlatformWebViewFragment.java

```diff
@@ -523,7 +523,7 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
       // Return to the previous activity if user has signed out
       String queryString = uri.getQuery();
       if (queryString != null && queryString.contains("portal:action=Logout")) {
-        clearWebviewData();
+        clearWebViewData();
```

Comment:
Could we call clearWebViewData(); inside mListener.onUserSignedOut(); , WDYT ?

Reply:
Done